src/test/java/picard/analysis/CollectWgsMetricsTest.java

```diff
@@ -239,10 +300,49 @@ public void testLargeIntervals(final String useFastAlgorithm) throws IOException
         }
     }
 
+    @Test(dataProvider = "wgsDataProvider")
+    public void testSmallIntervals(final File input, final String reference_name,
+                                   final String useFastAlgorithm) throws IOException {
+        final File outfile = File.createTempFile("testSmallIntervals", ".wgs_metrics",TEMP_OUTPUT_DIR);
+        final File ref = new File(reference_name);
+        final File intervals = new File(TEST_DIR, "smallIntervals.interval_list");
+        final int sampleSize = 1000;
+        final String[] args = new String[] {
```

Comment:
unnecessary array creation
```suggestion
        final String[] args = {
```

Reply:
🤯